tools/metadata/src/constants.ts

```diff
@@ -251,18 +251,21 @@ export const rinkebyTokensByAddress: Record<string, NormalizedToken> = {
     address: '0x5592ec0cfb4dbc12d3ab100b257153436a1f0fea',
     decimals: 18,
     symbol: 'DAI',
+    kind: 'ERC20',
```

Comment:
Would be `0x36372b07 ` but can just use `tokenKinds.ERC20` from constants

Reply:
This is meant to be a string representation either `'ERC20'` | `'ERC721'` to better match the schema used in the legacy hosted metadata service. It's used by the UI and by AirSwap.js to determine how a piece of token metadata should be treated